### PR TITLE
added IBInspectable thumb border width/color options

### PR DESCRIPTION
**Pull request by natecraft1**

### CODE REVIEW

**Comment by EranBoudjnah** [comment]
Tests seem to fail?

File: MTCircularSlider/Classes/MTCircularSlider.swift [hunk 1]
```diff
@@ -46,6 +46,8 @@ public enum Attributes {
 	case thumbRadius(CGFloat)
 	case thumbShadowRadius(CGFloat)
 	case thumbShadowDepth(CGFloat)
+    case thumbBorderWidth(CGFloat)
```
Comment (EranBoudjnah) [inline MTCircularSlider/Classes/MTCircularSlider.swift]: Indentation is wrong throughout.

Comment (natecraft1) [reply MTCircularSlider/Classes/MTCircularSlider.swift]: in my xcode the indentation looks correct. maybe my indentation preferences have to match yours? Not sure

Comment (EranBoudjnah) [reply MTCircularSlider/Classes/MTCircularSlider.swift]: I left tab settings at their default configuration. You use spaces to indent.

**Comment by EranBoudjnah** [comment]
Merged. However, I'd suggest another PR for updating the readme file.

**Pull request merged**